test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -134,16 +134,26 @@ class test_uct_sockaddr : public uct_test {
                          << " Interface: " << GetParam()->dev_name;
     }
 
+    size_t iface_priv_data_do_pack(void *priv_data)
+    {
+        size_t priv_data_len;
+
+        client_priv_data = "Client private data";
```

Comment:
class members should start with `m_` prefix

Reply:
let's leave the params in the test_uct_sockaddr class as is... it's planned to be removed anyway.

Reply:
> it's planned to be removed anyway.

ok then